ocs_ci/deployment/deployment.py

```diff
@@ -526,22 +526,6 @@ def deploy_ocs(self):
             selector='app=rook-ceph-tools', resource_count=1, timeout=600
         )
 
-        # Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1847098
```

Comment:
I see that the BZ is fixed in 4.5, but what about 4.4?

Reply:
I would maybe move this WA removal to another independent PR. WDYT?

Reply:
@petr-balogh I just opted to remove this here because I already have one open PR that is blocked by this one. I can revert this here and open another PR though. This one will just be blocked until the new one is merged.

Reply:
This has been merged in #2557